lib/mongoid/changeable.rb

```diff
@@ -94,6 +95,10 @@ def previous_changes
       @previous_changes ||= {}
     end
 
+    def previous_attributes
```

Comment:
Is this part of public API? If yes it needs a docstring and I suspect the mutation is problematic in the following scenario:

```
band = Band.create!
band.save!
attrs = band.previous_attributes
band.name = 'foo'
band.save! # does this mutate attrs I obtained earlier?
```

Reply:
I suppose better example would be

```
band = Band.create!
band.save!
attrs = band.previous_attributes
band.reload # does this mutate attrs I obtained earlier?
```

Reply:
See my answer to the comment above. In addition, I made this method `private`, and added a docstring.